### DISCUSSION TITLE
gdl rpm package available for openSUSE

### DISCUSSION THREAD

**Discussion by gcomes** [gnudatalanguage/gdl#1641] [General]
Since about a year and half, the gdl rpm package is actively maintained for openSUSE.
Download url: https://software.opensuse.org//download.html?project=science&package=gdl-language
May be the URL could be added to the wiki page: https://github.com/gnudatalanguage/gdl/wiki/GDL-on-Linux

**Comment by brandy125** [comment]
Nice to know.

Thank you!

**Comment by GillesDuvert** [comment]
Thank you very much, the information is now in the wiki!

**Comment by GillesDuvert** [comment]
silly of me to close this conversation
